main.go

```diff
@@ -70,6 +70,9 @@ func main() {
 		esExportIndicesSettings = kingpin.Flag("es.indices_settings",
 			"Export stats for settings of all indices of the cluster.").
 			Default("false").Envar("ES_INDICES_SETTINGS").Bool()
+		esExportIndicesSearchGroups = kingpin.Flag("es.indices_search_groups",
+			"Export search stats of all indices of the cluster by group.").
+			Default("false").Envar("ES_INDICES_TAGS").Bool()
```

Comment:
We've stopped using ENV vars here.

```suggestion
			Default("false").Bool()
```